### ISSUE TITLE
Issue while Importing Magic Wallet from @magic-sdk/admin --version ^1.8.0

### ISSUE THREAD

**Issue by Rafia10** [magiclabs/magic-admin-js#101]
✅ Prerequisites

- ✅  Did you perform a cursory search of open issues? Is this bug already reported elsewhere? 
- ✅  Are you running the latest SDK version? 
- ✅ Are you reporting to the correct repository (`@magic-sdk/admin`)? 

### 🐛 Description
While Importing Magic from admin-sdk an Error occurs.

### 🧩 Steps to Reproduce

1. Install magic-admin sdk
2. Import Magic Link in your working file and create magic provider 
3. Run Your project to test it

### 🤔 Expected behavior
We expected that our connection will be successfully build with magic link.

### 😮 Actual behavior

Instead of successful connection with magic link we have got this import error:
SyntaxError: The requested module '../node_modules/@magic-sdk/admin/dist/esm/index.js' does not provide an export named 'Magic'

[0]     at ModuleJob._instantiate (node:internal/modules/esm/module_job:128:21)
[0]     at async ModuleJob.run (node:internal/modules/esm/module_job:194:5)
[0]     at async Promise.all (index 0)
[0]     at async ESMLoader.import (node:internal/modules/esm/loader:385:24)
[0]     at async loadESM (node:internal/process/esm_loader:88:5)
[0]     at async handleMainPromise (node:internal/modules/run_main:61:12)
### 💻 Code Sample
https://gitlab.com/rafia.qadir/magic-link-issues/-/tree/main
### 🌎 Environment

| Software           | Version(s) |
| ------------------ | ---------- |
| `@magic-sdk/admin` | ^1.8.0
| Node               | v16.15.0
| Operating System   | ~20.04.1-Ubuntu

**Comment by keanemind** [comment]
Likely related to https://github.com/magiclabs/magic-admin-js/issues/85

**Comment by raymondfeng** [comment]
I ran into a similar issue:

```
(node:99465) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
/node_modules/@magic-sdk/admin/dist/esm/index.js:1
export { MagicAdminSDK as Magic } from './core/sdk';
^^^^^^

SyntaxError: Unexpected token 'export'
```

**Comment by maximeclaisse** [comment]
Same here. Would be great to fix, as this bug has been around for more than a year.

**Comment by CryptoGraffe** [comment]
Same problem here. Can't use w/ pure ESM+Typescript

**Comment by am-hernandez** [comment]
Hi @Rafia10 , cc @CryptoGraffe @maximeclaisse @raymondfeng @the-dijkstra @keanemind

We have released an update to the Admin SDK that resolves this issue. The fix is in version `2.1.0`. 

Please give it a try at your earliest convenience!
